scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -44,6 +44,8 @@ else
     exit 1
 fi
 
+git log -n 5
+
 # assert that kam is on the path
 kam version
```

Comment:
```
Login to the cluster as a developer user
+ git log -n 5
commit dc909af9e8d2bd4474a5dbecf4ad1517fb76d985
Merge: 337dae1 d7f8e42
Author: ci-robot <ci-robot@openshift.io>
Date:   Tue Dec 1 07:00:14 2020 +0000

    Merge commit 'd7f8e426a16b16a3aa3f34b5d54b067809a5e027'
```

The latest branch commit SHA is ```dc909af9e8d2bd4474a5dbecf4ad1517fb76d985```
```
+ kam version
kam version v0.0.14-15-gdc909af
```
@bigkevmcd 
Seems kam version capturing the correct SHA that is ```commit dc909af9e8d2bd4474a5dbecf4ad1517fb76d985```. However I am not sure why the latest commit has two SHA that is  ```Merge commit 'd7f8e426a16b16a3aa3f34b5d54b067809a5e027'``` and ```Merge commit 'd7f8e426a16b16a3aa3f34b5d54b067809a5e027'``` ?